src/Common/src/System/Net/WebHeaderEncoding.cs

```diff
@@ -75,57 +75,5 @@ internal static unsafe void GetBytes(string myString, int charIndex, int charCou
                 }
             }
         }
-        internal static unsafe byte[] GetBytes(string myString)
```

Comment:
This is used here:
https://github.com/dotnet/corefx/blob/5a37188238e90d8566e6161b43ac483a671bb7d2/src/System.Net.HttpListener/src/System/Net/Managed/HttpListenerResponse.Managed.cs#L281